pkg/cosign/errors.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cosign
+
+import "fmt"
+
+var (
+	// ErrNoMatchingSignatures is the error returned when there are no matching
+	// signatures during verification.
+	ErrNoMatchingSignatures = &VerificationError{"no matching signatures"}
+
+	// ErrNoMatchingAttestations is the error returned when there are no
+	// matching attestations during verification.
+	ErrNoMatchingAttestations = &VerificationError{"no matching attestations"}
```

Comment:
This is a good pattern, then we can start adding things like if the issuer/subject or other things are not checking out.